lib/get-s3-api.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _ = require('ls-lodash'),
+    aws = require('aws-sdk'),
+    s3Config = {};
```

Comment:
bonus, if you install aws cli it will ask you for creds and store them for you IIRC